vocab-importer/vocab_importer/main.py

```
@@ -0,0 +1,107 @@
+
+@app.post("/generate")
+async def generate_vocab(prompt: Optional[str] = Form(None)):
+    result = generate_vocab_with_openai(prompt)
+    if "error" in result:
+        return JSONResponse(content=result, status_code=400)
+    vocab_data.extend(result.get("groups", []))
+    return RedirectResponse(url="/", status_code=303)
+
+
+@app.post("/import")
+async def import_vocab(file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)